README.md

```diff
@@ -43,22 +43,20 @@ conda install -c conda-forge openjdk=21 maven -y
 pip install -r requirements.txt
 ```
 
-If building `nmslib` failed during installation, try manually installing the library with `conda install -c conda-forge nmslib` and following it up with `pip install -r requirements.txt` again.
```

Comment:
The most recent version of Pyserini no longer has nmslib as a mandatory dependency (because they realized the issue too lol), so this instruction is no longer relevant.

Reply:
We can also close issue #129 now